README.rst

```diff
@@ -53,7 +59,7 @@ or as a directory — and you want to distribute it.
        [project.urls]
        Home = "https://github.com/sirrobin/foobar"
 
-   You can edit this file to add other metadata, for example to set up
+   You can edit this file to add other metadata, for example, to set up
```

Comment:
I wouldn't add this comma since the example is setting up command line scripts, not the addition of other metadata.